<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>604</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ID GENERATOR</string>
  </property>
  <widget class="QLabel" name="head">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UID GENERATOR [QR CODE]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Full Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>43</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>147</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>82</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blood Group</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>117</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mobile No.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gender">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="dob">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="companyName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="age">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="address">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="blood">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="number">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Browse Image</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="imgtext">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>image</zorder>
  <zorder>head</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>gender</zorder>
  <zorder>dob</zorder>
  <zorder>companyName</zorder>
  <zorder>age</zorder>
  <zorder>pushButton</zorder>
  <zorder>address</zorder>
  <zorder>username</zorder>
  <zorder>blood</zorder>
  <zorder>number</zorder>
  <zorder>browse</zorder>
  <zorder>imgtext</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
